docs/processing_data.rst

```diff
@@ -152,6 +173,235 @@ Although this is a simple example of labeling data other things can be done here
 
 This example used a PI Server as the destination, however the same mechanism and filters may be used for any north destination.
 
+Graphical Pipeline Development
+------------------------------
+
+In versions 1.0 up until version 2.4 of Fledge the user interface for creating and editing pipeline was a purely grid based interface as illustrated above. In version 2.4 an option was introduced that allowed a more graphical approach to creating and managing data pipelines. This option was activated via the *Settings* menu option and is call the *Flow editor*. Enabling this will change the user interface for pipeline management.
+
+.. note::
+
+   As of version 3.0 the graphical, flow editor will be the default user interface mode. The previous interface is still available and may be selected by turning off the flow editor in the Settings menu.
+
+Adding A Pipeline
+~~~~~~~~~~~~~~~~~
+
+Adding a pipeline, in for example the south, is done by navigating to the *South* menu item, a page will appear that shows all of the current south service, or pipeline, that have been configured.
```

Comment:
to the *South* menu item, a page will appear -> navigating to the *South* menu item. A page will appear